core/modules/date/date.admin.inc

```diff
@@ -492,7 +492,7 @@ function _date_field_settings_form($field, $instance, $has_data) {
   }
   $tz_handling = $settings['tz_handling'];
 
-  $description = t('Select the date attributes to collect and store.');
+  $description = t('');
   if ($has_data) {
     $description .= ' ' . t('Changes to date attributes only effects new or updated content.');
```

Comment:
@Wylbur I think we should remove the `' ' . ` part of this now that the description is an empty string.

Reply:
So just change it to empty function 
```t();```

Reply:
on line 497; remove the first part that adds a space and concatenates; so it will just be 

```php
$description .= t('Changes to date attributes only effects new or updated content.');
```

Reply:
@serundeputy I changed to empty t(); but now it fails tests.

Reply:
@Wylbur you can remove the emtpty `t()` and instead just set the varaible to an empty string, like this:
```
$description = '';
if ($has_data) {
  $description = t('..');
}
```